hunting/hunting.py

```diff
@@ -84,6 +84,9 @@ async def leaderboard(self, ctx, global_leaderboard=False):
         async with ctx.typing():
             sorted_acc = sorted(userinfo.items(), key=lambda x: (x[1]["total"]), reverse=True)[:50]
 
+        if not hasattr(ctx.guild, "members"):
```

Comment:
```py
# not sure this is possible too, but, it was here before (check for AttributeError), so... Yes?
```

Reply:
It shouldn't be possible since this is a guild only command but no harm in leaving it in there I suppose.